src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -45,6 +46,18 @@ public ManagementAPI(String domain, String apiToken) {
                 .build();
     }
 
+    /**
+     * Update the API token to use on new calls. This is useful when the token is about to expire or it already has.
```

Comment:
"when the token is about to expire or already has"

Reply:
I had doubts on this one 😛